packages/core/src/modules/proofs/ProofsApi.ts

```diff
@@ -70,6 +70,11 @@ export interface ProofsApi<PFs extends ProofFormat[], PSs extends ProofService<P
     options: AutoSelectCredentialsForProofRequestOptions
   ): Promise<FormatRequestedCredentialReturn<PFs>>
 
+  // Get Requested Credentials
+  getRequestedCredentialsForProofRequest(
+    options: AutoSelectCredentialsForProofRequestOptions
```

Comment:
Can we define a custom type for this? This type is meant for the auto select method